README.md

```diff
@@ -28,31 +28,134 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## API Key
```

Comment:
This 'API Key' section seems sparse from a usability perspective. Consider adding information on how to get at their API key after a `planet init` as well as information on how to set that as an environment variable for use in the following library example, e.g. `export PL_API_KEY=$(jq --raw-output '.key' $HOME/.planet.json)` and maybe the equivalent for `setx` or PowerShell.

Reply:
This is a great point. We have functionality for storing the API key via the CLI but no functionality for getting it. The primary use case for pulling the api key from the environmental variable is for use in a Docker container (as done in [planet notebooks](https://github.com/planetlabs/notebooks)). The api key is passed as a command-line argument when the Docker container is built. What about adding a CLI command for retrieving the api key so users and docs don't have to hard-code the path to the planet auth information?

So CLI commands for interacting with auth would be e.g. 
storing api key
`planet auth --init`

retrieving api key
 `planet auth --key`

usage example:
`export PL_API_KEY=$(planet auth --key)`

Reply:
That sounds great to me! My vote would be for `planet auth --key` to explode with a non-zero return if it's unset, similar to `set +u` or `set +o nounset`.

Reply:
Awesome! This work will be done in #252